blocks/magazine-hero/magazine-hero.js

```diff
@@ -1,5 +1,7 @@
 import { createOptimizedPicture, decorateIcons, readBlockConfig } from '../../scripts/lib-franklin.js';
 
+const darkColor = '#3d3d3d';
```

Comment:
try to use a CSS variable instead of JS variable. example
```
block.style.setProperty('--accent-color', `var(--color-${accentColor})`);
```

Reply:
awesome I dint know we could do this..